### ISSUE TITLE
Package yggdrasil for Winget

### ISSUE THREAD

**Issue by Mikaela** [yggdrasil-network/yggdrasil-go#713]
[Winget](https://github.com/microsoft/winget-cli/) is Microsoft's own package manager under development, see [the link for instructions getting the preview](https://github.com/microsoft/winget-cli/).

* Winget package repository: https://github.com/microsoft/winget-pkgs

Loosely related to https://github.com/yggdrasil-network/yggdrasil-go/issues/611 in case Chocolatey itself gets packaged (https://github.com/microsoft/winget-pkgs/pull/367).

**Comment by majestrate** [comment]
i didn't know winget did package management yet, last i checked it was just an exe downloader with no ability to manage (remove, depend, upgrade, migrate, etc) packages.

**Comment by Mikaela** [comment]
Sadly that is still the case, for now. https://github.com/microsoft/winget-cli/issues/223

**Comment by ajgrf** [comment]
Maybe it's time to revisit this. Winget is still just a wrapper for installer EXEs and MSIs, but upgrades and removals seem to work pretty well now. Well enough that I'm now using winget for almost everything that I used to need chocolatey for.

**Comment by pomazanbohdan** [comment]
Winget has recently become very strong and has become the main way for me to work with software, even on servers, please add it to the repository.

**Comment by pomazanbohdan** [comment]
> i didn't know winget did package management yet, last i checked it was just an exe downloader with no ability to manage (remove, depend, upgrade, migrate, etc) packages.

Winget has recently become very strong and has become the main way for me to work with software, even on servers, please add it to the repository.

**Comment by neilalexander** [comment]
Anyone who wants to volunteer to support the Windows package would be welcome to do so.

**Comment by pomazanbohdan** [comment]
Yes, but it would be good if it was done by the community yggdrasil and not by a third party, so as not to lose control and automate it later.

P.S. 

https://github.com/microsoft/winget-create

**Comment by SNMetamorph** [comment]
> Anyone who wants to volunteer to support the Windows package would be welcome to do so.

Ok, why not
https://github.com/microsoft/winget-pkgs/pull/135597

**Comment by SNMetamorph** [comment]
Now Yggdrasil can be installed on Windows with just one line: `winget install YggdrasilNetwork.Yggdrasil`
So I think we can close this issue and also update installation instructions.

**Comment by lostmsu** [comment]
It's quite outdated right now.

**Comment by SNMetamorph** [comment]
So, I tried to keep package up-to-date, but unfortunately that's seemed almost impossible because of weird Microsoft malware checking false-positives.

**Comment by neilalexander** [comment]
Unfortunately this is an extremely common problem with Go binaries and Windows Defender. I have never found a good solution that doesn't involve expensive code signing. For a while I was submitting builds as false positives to Microsoft but the next one would always end up getting flagged anyway. 

If it isn't feasible to keep the package up-to-date then it is probably best to withdraw it from Winget completely, otherwise users run the risk of installing and running old versions without realising it.

**Comment by SNMetamorph** [comment]
We were also extremely annoyed with our Xash3D FWGS engine being false-positively detected as malware by Microsoft Defender. Once we added signing binaries with self-signed certificate, this somehow helped and we didn't received any reports about false-positive detections since then. So this may be an option too.
